tensorflow_datasets/core/dataset_info.py

```diff
@@ -265,6 +268,29 @@ def _dataset_info_path(self, dataset_info_dir):
   def _license_path(self, dataset_info_dir):
     return os.path.join(dataset_info_dir, LICENSE_FILENAME)
 
+  def visualize_dataset(self):
+    return self._visualize_dataset(self._builder, 'train')
+
+  def _visualize_dataset(self, builder, split):
+    """Visualize dataset statistics with FACETS DIVE."""
+    filetype_suffix = builder._file_format_adapter.filetype_suffix  # pylint: disable=protected-access
+    if filetype_suffix not in ["tfrecord", "csv"]:
```

Comment:
We dropped support for csv. You can remove.

Reply:
Oky!